docs/deployment-guide-bicep.md

```diff
@@ -97,7 +97,10 @@ Parameter name | Default Value | Description
 -------------- | ------------- | -----------
 `deployDefender` | 'false' | When set to "true", enables Microsoft Defender for Cloud for the subscriptions used in the deployment. It defaults to "false".
 `emailSecurityContact` | '' | Email address of the contact, in the form of john@doe.com
+`pricingTier` | 'Standard' | The standard pricing tier offers advanced security capabilities
```

Comment:
`pricingTier` is not a parameter for mlz.bicep, so it does not belong in this table. (It is a parameter for the resource provider called by MLZ, but someone using MLZ would not set that parameter.)

Reply:
Updated in commit [4c7a06c](https://github.com/Azure/missionlz/pull/687/commits/4c7a06c301a8d9b9e54b2e72a4a58644c3607fbb)
resolving